optimum/exporters/openvino/model_configs.py

```diff
@@ -3887,6 +3884,15 @@ def patch_model_for_export(
         return DeepseekPatcher(self, model, model_kwargs=model_kwargs)
 
 
+@register_in_tasks_manager(
+    "deepseek_v3", *["text-generation", "text-generation-with-past"], library_name="transformers"
+)
+class DeepseekVOpenVINOConfig(LlamaOpenVINOConfig):
```

Comment:
before we can merge : need to fix `rope_interleave` for export https://github.com/huggingface/transformers/blob/v4.55.4/src/transformers/models/deepseek_v3/modeling_deepseek_v3.py#L402

Reply:
cc @openvino-dev-samples any news on this on your side ?